discovery-client/src/cli.ts

```
@@ -47,10 +49,9 @@ const createLogger = (argv: Argv): Logger => {
   }
 }
 
-const debugLogArgvMiddleware = (argv: Argv) => {
+const debugLogArgvMiddleware: MiddlewareFunction<Argv> = (argv): void => {
   const log = createLogger(argv)
   log.debug(`Calling ${argv.$0} with argv:`, argv)
-  return argv
```

Throw a TODO and ignore on here instead of removing this return